### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by vnky457**
need to be save for today work review

### CODE REVIEW

File: mailfinal [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+I'm regretted to inform you like this, I'm not able to take HCL opportunity forward. I got offer to From Malaysia it's like permanent position.
+
+About this I informed to Ms.Sushmitha from Cadmaxx. 
+
+It's really sorry for the inconvenience.
+
+If possible, can you stop the processing sending Laptop to me.
+
+For exiting formalities , Ms.Sushmitha mention me like contact reporting manager
```
Comment (vnkyperumalla) [inline mailfinal:11]: need to add thank you and Regards

**Review by vnkyperumalla — CHANGES_REQUESTED** [review]
only one reqeust change

File: mailfinal (hunk 1 above)
Comment (vnky457) [reply mailfinal:11]: Sure Thank you.

Comment (vnkyperumalla) [reply mailfinal:11]: Okay

**Review by vnkyperumalla — APPROVED** [review]

**Pull request merged**